Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM cimg/python:3.11.1-node
+FROM cimg/python:3.12.0-node
```

Comment:
Why does the base image use Python 3.12 when Sceptre so far only supports 3.11? Does something else need 3.12?

Reply:
Sceptre does not need python 3.12 at this time since it's not supported yet.  We are adding 3.12 here so that we can flip the switch in [Sceptre tox.ini](https://github.com/Sceptre/sceptre/blob/master/tox.ini#L3) to make 3.12 support happen.

Reply:
That makes sense.

Why do we use the node variant of the image?

Reply:
> Why do we use the node variant of the image?

i don't know other than `it works`.